### ISSUE TITLE
[a11y] 9.2.4.1 - Skip areas

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5391]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
- **Pages 1 to 3:** it is recommended to only use one header element on the site. See https://dequeuniversity.com/rules/axe/4.2/landmark-no-duplicate-banner?application=AxeChrome for more details.

## Status: rather fulfilled
- **Page 2 (Mit anderen geteilten Dateien)**
See general notes

- **Page 3 (Gelöschte Dateien)**
See general notes

- **Page 4 (Konto)**
The content area of the site is not marked up with a main-landmark.

## Status: partly fulfilled
- **Page 1 (Alle Dateien)**
Copy or move elements: the breadcrum navigation was correctly implemented with a navelement but it has no label. For labelling an aria-label attribute could be used.
See general notes

**Comment by pascalwengerter** [comment]
Needs a fix in the ODS

**Comment by stevefrenzel** [comment]
@tbsbdr @kulmann 

I would suggest replacing the `header` element of the sidebar with a `figure` element. The header element on the individual pages makes more sense than in the sidebar, unless this leads to not meeting an accessibility requirement.

Alternatively a `div` with a role of "img". In it's core it's a link, but first and foremost an image, so this would solve the issue. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role

**Comment by pascalwengerter** [comment]
As pointed out in #6007 it may be we have to add dedicated `<main>` tags for the `pages` in the runtime - the runtime otherwise relies on the rendered app to provide a `<main>` tag since it doesn't take any guesses on how the rendered app is structured. Not sure if this is necessary for the A11y audit though

**Comment by pascalwengerter** [comment]
> As pointed out in #6007 it may be we have to add dedicated `<main>` tags for the `pages` in the runtime - the runtime otherwise relies on the rendered app to provide a `<main>` tag since it doesn't take any guesses on how the rendered app is structured. Not sure if this is necessary for the A11y audit though

Not sure what I was going for here, feel free to ignore. Just merged a PR in ODS (https://github.com/owncloud/owncloud-design-system/pull/1773) that should resolve what is pointed out in `general notes`, thanks to @stevefrenzel
